packages/docs/src/routes/docs/menu.md

```diff
@@ -16,10 +16,29 @@
 - [Context](</docs/(qwik)/components/context/index.mdx>)
 - [Slots](</docs/(qwik)/components/slots/index.mdx>)
 - [Rendering](</docs/(qwik)/components/rendering/index.mdx>)
-- [Styling](</docs/(qwik)/components/styles/index.mdx>)
+- [Styling][def]
```

Comment:
?

Reply:
I had previously added on last line of code will be updated in my upcoming commit !